test/minikube/minikube_base_restore_test.go

```diff
@@ -436,3 +435,91 @@ func TestKubernetesAutoRestore(t *testing.T) {
 		testlib.CheckArchives(t, namespaceName, admin0, opt.DbName, 2, 0)
 	})
 }
+
+// Basic test for creating a database off of a VolumeSnapshot
+func TestKubernetesSnapshotRestore(t *testing.T) {
+	testlib.AwaitTillerUp(t)
+	defer testlib.VerifyTeardown(t)
+
+	options := helm.Options{}
+
+	defer testlib.Teardown(testlib.TEARDOWN_ADMIN)
+
+	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &options, 1, "")
+
+	kubectlOptions := k8s.NewKubectlOptions("", "", namespaceName)
+	admin := fmt.Sprintf("%s-nuodb-cluster0", helmChartReleaseName)
+	admin0 := fmt.Sprintf("%s-0", admin)
+
+	testlib.AddDiagnosticTeardown(testlib.TEARDOWN_ADMIN, t, func() {
+		k8s.RunKubectl(t, kubectlOptions, "get", "pods", "-o", "wide")
+		testlib.DescribePods(t, namespaceName, admin)
+	})
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE)
+
+	sourceDatabaseChartName := testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
+		SetValues: map[string]string{
+			"database.sm.resources.requests.cpu":                       testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory":                    testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":                       testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory":                    testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.persistence.storageClass":                        testlib.SNAPSHOTABLE_STORAGE_CLASS,
+			"database.sm.hotCopy.journalPath.persistence.storageClass": testlib.SNAPSHOTABLE_STORAGE_CLASS,
```

Comment:
I was trying to minimize the amount of Helm values I was passing in to only those strictly necessary for the test to run. If I move away from that, it is better to just use `noHotCopy` SMs.

Reply:
What was the resolution here? This is still HCSM and cronjobs are still enabled.

It makes sense to disable cronjobs whenever they are not needed because they may unexpectedly generate pods that cause the "are all NuoDB pods gone?" checks to fail.

Reply:
I changed the long tests to use non-HC SMs. I did not do the same for the short test. I will do so.